doc/DeveloperGuide.md

```diff
@@ -88,6 +88,27 @@ Use case ends.
 > 3a1. AddressBook shows an error message <br>
   Use case resumes at step 2
 
+#### Use case: Rename tag
+
+**MSS**
+
+1.User requests to view tags
+2.AddressBook shows a list of person's tags.
+3.User request to change specific tag.
+4.AddressBook changes all of the same specific tags to requested tags.
```

Comment:
Perhaps you can consider addressbook prompting for confirmation from a usability standpoint?